redis-om-spring/src/main/java/com/redis/om/spring/RedisEnhancedKeyValueAdapter.java

```diff
@@ -225,19 +225,17 @@ public <T> T delete(Object id, String keyspace, Class<T> type) {
    */
   @Override
   public void deleteAllOf(String keyspace) {
-    Class<?> type = keyspaceToIndexMap.getEntityClassForKeyspace(keyspace);
-    List<byte[]> keys = getAllIds(keyspace, type).stream().map(id -> getKey(keyspace, id)).map(k -> toBytes(k))
-        .collect(Collectors.toList());
-    if (keys.size() > 0) {
-      redisOperations.execute((RedisCallback<Void>) connection -> {
-        connection.del(keys.toArray(new byte[0][]));
-        return null;
-      });
+    Class<?> type = indexer.getEntityClassForKeyspace(keyspace);
+    Optional<String> maybeSearchIndex = indexer.getIndexName(keyspace);
+    if (maybeSearchIndex.isPresent()) {
+      SearchOperations<String> searchOps = modulesOperations.opsForSearch(maybeSearchIndex.get());
+      searchOps.dropIndex();
```

Comment:
Is this using the `FT.DROP`? or `FT.DROPINDEX DD` - I'd use the non-deprecated command.

Reply:
It's mapped to `DROP("FT.DROP"),` in the Jedis version I'm stuck with there is no `FT.DROPINDEX` afaik